server/controllers/participantsController/index.js

```diff
@@ -7,7 +7,7 @@ const ParticipantsController = {
   index: async (req, res) => {
     try {
       const { dataDb, appDb } = req.app.locals
-      const parsedQueryParams = qs.parse(req.query) || DEFAULT_PARTICIPANTS_SORT
+      const parsedQueryParams = Object.keys(req.query).length ? qs.parse(req.query) : DEFAULT_PARTICIPANTS_SORT
```

Comment:
I think `req.query` is a string, no? I believe `qs.parse(req.query)` converts it into an object.

Reply:
In my console it was defaulting to an object. I don't think `qs.parse` is necessary?

Okay, did some digging and it's not necessary because of this middleware option:

Reply:
[e670c31](https://github.com/AMP-SCZ/dpdash/pull/413/commits/e670c31999e59e2e361b22f1d3f547a4b1d117b6)